### PR TITLE
Fix: bug of reset password

### PR DESCRIPTION
**Pull request by SirineMhedhbi**
## Context:
see: https://github.com/ontoportal-lirmm/bioportal_web_ui/issues/276

## Changes:
- fix bug after clicking on the reset password link sent by email

### CODE REVIEW

**Review by syphax-bouazzouni — CHANGES_REQUESTED** [review]
Do a PR description

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -250,7 +250,7 @@ def validate_update(params)
     if params[:username].nil? || params[:username].length < 1
       errors << "Last name field is required"
     end
-    if ((!params[:orcidId].match(/^\d{4}+(-\d{4})+$/)) || (params[:orcidId].length != 19)) && !(params[:orcidId].nil? || params[:orcidId].length < 1)
+    if params[:orcidId].present? && ((!params[:orcidId].match(/^\d{4}-\d{4}-\d{4}-\d{4}$/)) || (params[:orcidId].length != 19))
```
Comment (syphax-bouazzouni) [inline app/controllers/users_controller.rb:253]: you changed the REGEX, why ?

**Review by syphax-bouazzouni — CHANGES_REQUESTED** [review]

**Comment by SirineMhedhbi** [comment]
@syphax-bouazzouni  
Already exists the description :)

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (SirineMhedhbi) [reply app/controllers/users_controller.rb:253]: because a valid ORCID ID consists of 16 digits and 3 dashes

Comment (syphax-bouazzouni) [reply app/controllers/users_controller.rb:253]: @Bilelkihal please review this.

**Pull request merged**